commandapi-core/src/main/java/dev/jorel/commandapi/InternalConfig.java

```diff
@@ -65,6 +67,7 @@ public InternalConfig(CommandAPIConfig<?> config) {
 		this.verboseOutput = config.verboseOutput;
 		this.silentLogs = config.silentLogs;
 		this.useLatestNMSVersion = config.useLatestNMSVersion;
+		this.beMoreLenientForMinorVerions = config.lenientForMinorVersions;
```

Comment:
Small note: it's a bit strange these fields have different names, which isn't the case for other config options.